package.json

```diff
@@ -21,7 +21,6 @@
     "@flipt-io/flipt-client-browser": "^0.0.17",
     "@growthbook/growthbook": "^1.0.0",
     "@grpc/grpc-js": "^1.9.13",
-    "@openfeature/ofrep-core": "^0.1.4",
```

Comment:
We shouldn't have any deps that are local to this repo (dev or otherwise). We always want to resolve the local ones.